cnf-certification-test/operator/suite.go

```diff
@@ -99,28 +107,40 @@ func testOperatorInstallationWithoutPrivileges(env *provider.TestEnvironment) {
 						env.Operators[i], permission.ServiceAccountName, n, ruleIndex)
 					// Keep reviewing other permissions' rules so we can log all the failing ones in the claim file.
 					badRuleFound = true
+					nonCompliantObjects = append(nonCompliantObjects, testhelper.NewOperatorReportObject(env.Operators[i].Namespace, env.Operators[i].Name, "Operator has privileges on cluster resources ", false).
+						AddField(testhelper.ServiceAccountName, permission.ServiceAccountName).AddField(testhelper.ResourceName, strings.Join(permission.Rules[ruleIndex].ResourceNames, "")))
+				} else {
+					compliantObjects = append(compliantObjects, testhelper.NewOperatorReportObject(env.Operators[i].Namespace, env.Operators[i].Name, "Operator has no privileges on cluster resources", true).
+						AddField(testhelper.ServiceAccountName, permission.ServiceAccountName).AddField(testhelper.ResourceName, strings.Join(permission.Rules[ruleIndex].ResourceNames, "")))
```

Comment:
strings.Join(permission.Rules[ruleIndex].ResourceNames, this is always empty. Replace with "n/a" or similar

add "s" for plural to testhelper.ResourceName -> testhelper.ResourceName+"s"